static/app/components/avatar/avatarList.tsx

```diff
@@ -13,14 +13,16 @@ const defaultProps = {
   tooltipOptions: {},
 };
 
+type UserAvatarProps = React.ComponentProps<typeof UserAvatar>;
```

Comment:
should we make the `UserAvatar` props exportable?

Reply:
I'm indifferent, we should probably clean that up later more consistently 🤷‍♂️